cli/command/container/opts.go

```diff
@@ -825,6 +832,23 @@ func parseSecurityOpts(securityOpts []string) ([]string, error) {
 	return securityOpts, nil
 }
 
+func parseSystemPaths(securityOpts []string) ([]string, []string, []string, error) {
+	var maskedPaths []string
+	var readonlyPaths []string
+	filtered := securityOpts[:0]
+	for _, opt := range securityOpts {
+		con := strings.SplitN(opt, "=", 2)
+		if con[0] == "systempaths" && con[1] == "unconfined" {
```

Comment:
Looks like we don't need the split (and don't strip whitespace etc) at this point, so this could be simplified to

```suggestion
		if opt == "systempaths=unconfined" {
```